test/e2e/atlas/search_test.go

```diff
@@ -223,4 +223,173 @@ func TestSearch(t *testing.T) {
 		expected := fmt.Sprintf("Index '%s' deleted\n", indexID)
 		assert.Equal(t, string(resp), expected)
 	})
+
+	t.Run("Create combinedMapping", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			"loadSampleData",
+			g.clusterName,
+			"--projectId", g.projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		_, err := cmd.CombinedOutput()
+		require.NoError(t, err)
+
+		fileName := fmt.Sprintf("create_index_search_test-%v.json", n)
+
+		file, err := os.Create(fileName)
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+		defer func() {
+			if e := os.Remove(fileName); e != nil {
+				t.Errorf("error deleting file '%v': %v", fileName, e)
+			}
+		}()
+
+		tpl := template.Must(template.New("").Parse(`
+{
```

Comment:
There is maintenance risk of that solution. 
In this case I think we are safe to use it, but in future we should avoid blobs like this as any change in api will break e2e

Reply:
Actually checking only mappings page would be much better as this is what we testing here.